### PR TITLE
Test index.xml

### PR DESCRIPTION
**Pull request by sugan89**

### CODE REVIEW

File: src/pe2loaddata/__main__.py [hunk 1]
```diff
@@ -69,8 +69,8 @@ def headless(
     # Strip spaces because XML parser is broken
     channels = dict([(str(k).replace(" ", ""), v) for (k, v) in channels.items()])
 
-    if not index_file:
-        index_file = os.path.join(index_directory, "Index.idx.xml")
+    if not index_file:       
+        index_file = os.path.join(index_directory, "Index.xml")
```
Comment (bethac07) [inline src/pe2loaddata/__main__.py:73]: This should be a glob to catch either Index or Index.idx, please and thank you!

Comment (sugan89) [reply src/pe2loaddata/__main__.py:73]: Sure, I will change it.  Thanks, Beth!

**Comment by sugan89** [comment]
The current code works with Index.idx.xml (Harmony's version 5) and Index.xml (Harmony's version 7). I am yet to try the XML files from Harmony's version 6.

**Comment by sugan89** [comment]
The current code also works on XML files from Harmony's version 6.

File: README.md [hunk 2]
```diff
@@ -15,38 +15,46 @@ To run CSV creation based on the XML file:
...
-The config.yml file lets you name the channels you want to save and lets you pull metadata out of the image. An example:
+The config.yml file lets you name the channels you want to save and lets you pull metadata out of the image. Based on the version of the `.xml` file, you can choose the `config_for_version5.xml` or `config_for_version6and7.yml` file from the `config` folder. An example of the `config.yml` if one is using the `.xml` file from version 5 of Harmony software:
 
     channels:
         HOECHST 33342: OrigDNA
         Alexa 568: OrigAGP
         Alexa 647: OrigMito
         Alexa 488: OrigER
         488 long: OrigRNA
+    channelid:
```
Comment (bethac07) [inline README.md:26]: We now don't need this line after the changes we made

File: config/config_for_version5.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+channels:
+    HOECHST 33342: OrigDNA
+    Alexa 568: OrigAGP
+    Alexa 647: OrigMito
+    Alexa 488: OrigER
+    488 long: OrigRNA
+channelid:
```
Comment (bethac07) [inline config/config_for_version5.yml:7]: We now don't need this line after the changes we made

**Pull request closed without merging**